packages/plugin-cache-memory/src/memory.ts

```diff
@@ -80,9 +85,19 @@ export default ({
                     memoryCacheOutdated: false,
                 });
 
+                const cashedValue = lruCache.get(cacheKey);
+                // when plugin break flow, plugin-http won't be called and meta will be empty,
+                // so we need to enrich the meta with status data, as it is done in a normal flow
+                if (cashedValue.status) {
+                    context.updateInternalMeta(metaTypes.PROTOCOL_HTTP, {
+                        response: {
+                            status: cashedValue.status
```

Comment:
question: it is expected that other response properties will be missing?

Also, do we need to add the same logic for `headers`?

Reply:
You are right, headers are also missing and it can be implemented in a similar way but storing all headers can hit memory consumption. @a-ryazanov what do you think about it ?

Reply:
We don't have plans for active maintaining this package, so i think we can skip storing headers for now.